libraries/chain/fork_database.cpp

```diff
@@ -666,15 +670,15 @@ namespace eosio::chain {
 
    template<class BSP>
    BSP fork_database_t<BSP>::get_block(const block_id_type& id,
-                                       check_root_t check_root /* = check_root_t::no */) const {
+                                       include_root_t include_root /* = include_root_t::no */) const {
```

Comment:
Remove the `/* = include_root_t::no */`

Reply:
Why? It provides doc on what the default is.

Reply:
Oh, I thought `include_root` was not a parameter with a default value. Don't make any changes for this PR. In general, should we stay away from default parameters?

Reply:
In general, I'm not a fan of default parameters. Here I was just changing the name.